src/Warehouse.java

```diff
@@ -26,34 +26,34 @@ public List<Product> getProductsInStock() {
         return devices;
     }
 
-//    public List<Product> filterByAttribute(String input, String typeResearched) {
```

Comment:
Nella mia pull request ho rimosso per sbaglio questo filtro commentato, prima di fare il merge a questa pull request assicuriamoci che venga mergiata la mia

Reply:
Questo è successo perchè ho sbagliato branch su cui pushare, pushando sul vecchio branch che era ancora presente in remoto